lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -178,7 +179,7 @@ export class AccountEntity {
         oboAssertion?: string
     ): AccountEntity {
         const account: AccountEntity = new AccountEntity();
-        
+
         account.authorityType = (authority.authorityType === AuthorityType.Adfs) ? CacheAccountType.ADFS_ACCOUNT_TYPE : CacheAccountType.GENERIC_ACCOUNT_TYPE;
         account.homeAccountId = idToken.claims.sub;
```

Comment:
Can we add `localAccountId` to the generic account generator as well?

Reply:
Done